crates/router/src/connector/powertranz/transformers.rs

```diff
@@ -239,6 +241,7 @@ pub struct PowertranzBaseResponse {
     iso_response_code: String,
     redirect_data: Option<String>,
     response_message: String,
+    reference: Option<String>,
```

Comment:
Same in response , the merchat reference would be order_identifier . 
```suggestion
    order_identifier : Option<String>,
```

Reply:
done